.github/workflows/release.yaml

```diff
@@ -454,7 +464,9 @@ jobs:
           .\wingetcreate.exe update Coder.Coder `
             --submit `
             --version "${version}" `
-            --urls "${installer_url}|X64" `
+            --urls "${x64_installer_url}|X64" `
+            --urls "${x64_zip_url}|X64" `
+            --urls "${arm64_zip_url}" `
```

Comment:
We do not `arm64` as `wingetcreate` can detect `arm64` from the URL for first two x64 links it can not.

Reply:
You should still specify it. We should not rely on this detection when all it takes is a few characters

Reply:
`wingetcreate` does not need architecture override anymore as it now supports parsing the URL for `amd64` and correctly marking it as x64 architecture. 

Reference: 
1. https://github.com/microsoft/winget-create/blob/08baf0e61e62dabcb2487397984fc69fad6a7499/src/WingetCreateCore/Common/PackageParser.cs#L594
2. PR: https://github.com/microsoft/winget-create/pull/445
2. This has been available since version https://github.com/microsoft/winget-create/releases/tag/v1.5.3.0